app/internal/page/donor/choose_attorneys_address.go

```diff
@@ -59,6 +59,7 @@ func ChooseAttorneysAddress(logger Logger, tmpl template.Template, addressClient
 			if data.Form.Action == "skip" {
 				attorney.Address = place.Address{}
 				lpa.Attorneys.Put(attorney)
+				lpa.Tasks.ChooseAttorneys = page.TaskCompleted
```

Comment:
this is a temporary fix because I messed it up in a previous PR, I've put a ticket in the backlog to actually sort all this logic out